### PR TITLE
 resolved MisconfigurationException

### PR DESCRIPTION
**Pull request by Qiliang**
end of epoch will throw execption:

pytorch_lightning.utilities.exceptions.MisconfigurationException: ModelCheckpoint(monitor='val_loss') not found in the returned metrics: ['loss', 'l_mle', 'l_length', 'logdet']. HINT: Did you call self.log('val_loss', value) in the LightningModule?

Signed-off-by: Qiliang <xiaoqlster@gmail.com>

### CODE REVIEW

**Review by blisc — APPROVED** [review]
Thanks for the fix!
